internal/api/config/config.go

```diff
@@ -88,6 +90,10 @@ func (c *Config) Verify() error {
 		return fmt.Errorf("encryption_key must be longer than 32 bytes")
 	}
 	c.EncryptionKey = promConfig.Secret(hex.EncodeToString([]byte(c.EncryptionKey)))
+	if c.ActivatePermission == nil {
+		var activatePermission = true
+		c.ActivatePermission = &activatePermission
```

Comment:
Not required but we could leverage this package "k8s.io/utils/ptr" and use the `ptr.To` 😄